From eec5928523cc1702e19f3f54ad1c798382a8c0c9 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Thu, 27 Jun 2024 09:53:57 +0200
Subject: [PATCH 0960/1695] plat-stm32mp2: remove unused CFG_STM32_FIREWALL
 configuration switch

Remove CFG_STM32_FIREWALL config switch that is now useless.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 83278eacfff1098e3de98ec341113086861259ae plat-stm32mp2: remove unused CFG_STM32_FIREWALL configuration switch]
Change-Id: I90b7044519a92a3ab697a00cf5777bf0e81570f3
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/390163
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/plat-stm32mp2/conf.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp2/conf.mk b/core/arch/arm/plat-stm32mp2/conf.mk
index 09b715c61..1e1554a92 100644
--- a/core/arch/arm/plat-stm32mp2/conf.mk
+++ b/core/arch/arm/plat-stm32mp2/conf.mk
@@ -100,7 +100,6 @@ $(call force,CFG_SCMI_SCPFW_PRODUCT,optee-stm32mp2)
 $(call force,CFG_SCMI_SERVER_REGULATOR_CONSUMER,y)
 $(call force,CFG_SECURE_TIME_SOURCE_CNTPCT,y)
 $(call force,CFG_STM32_SHARED_IO,y)
-$(call force,CFG_STM32_FIREWALL,y)
 $(call force,CFG_STM32_HSE_MONITORING,y)
 $(call force,CFG_STM32_PWR,y)
 $(call force,CFG_STM32_PWR_REGUL,y)
-- 
2.47.2

